src/shared/components/Button/component.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ButtonWrapper, ButtonVariant } from './styles';
+
+type ButtonProps = {
+  variant: ButtonVariant;
+  title: string;
+};
+
+const Button: React.FC<ButtonProps> = ({ variant, title }: ButtonProps) => {
+  return <ButtonWrapper variant={variant}>{title}</ButtonWrapper>;
+};
```

Comment:
Btw, if you don't have any side effects, you can write without `return`:
```javascript
const Button: React.FC<ButtonProps> = ({ variant, title }: ButtonProps) => ( <ButtonWrapper variant={variant}>{title}</ButtonWrapper>;
);
`

Reply:
fixed